erpnext/buying/doctype/request_for_quotation/request_for_quotation.js

```diff
@@ -56,21 +56,48 @@ frappe.ui.form.on("Request for Quotation",{
 
 	get_suppliers_button: function (frm) {
 		var doc = frm.doc;
+		console.log('s')
 		var dialog = new frappe.ui.Dialog({
 			title: __("Get Suppliers"),
 			fields: [
-				{	"fieldtype": "Select", "label": __("Get Suppliers By"),
+				{
+					"fieldtype": "Select", "label": __("Get Suppliers By"),
 					"fieldname": "search_type",
-					"options": "Tag\nSupplier Group", "reqd": 1 },
-				{	"fieldtype": "Link", "label": __("Supplier Group"),
+					"options": "Tag\nSupplier Group", "reqd": 1,
+					'onchange': function() {
+						if(dialog.fields_dict.search_type.value == 'Tag'){
+							frappe.call({
+								method: 'erpnext.buying.doctype.request_for_quotation.request_for_quotation.get_supplier_tag',
+								async: false,
```

Comment:
This is a bad pattern. Please avoid. This will block browser's main thread until the request is completed.

Reply:
done